### DISCUSSION TITLE
How it works..

### DISCUSSION THREAD

**Discussion by Joebel81** [raweceek-temeletry/f1-2021-udp#8] [Q&A]
Hello guys, i'd want ask sorry if my question could result funny for you all but i'm trying from 2 weeks to start with this program but at the moment i didn't take the hoped result.
I dowloaded the file zip in to may Laptop, then i started with 'npm run install' command and 'npm run start' too..
Now i can see in to the terminal (Mac) that i'm receiving the data from ump but i can't find it on no side!
Please may you help me?
<img width="433" alt="Schermata 2021-07-29 alle 14 14 42" src="https://user-images.githubusercontent.com/87584168/127489933-3b66daef-462f-4fc7-8edc-36404b504a2b.png">
<img width="779" alt="Schermata 2021-07-29 alle 14 15 53" src="https://user-images.githubusercontent.com/87584168/127490090-11a05ad8-8515-4607-9517-d4e4e5ea74dc.png">

**Comment by PrimeGoose** [comment]
`f1-2021-udp` is a library that can be leveraged to build your own telemetry application. `npm start` is used for development and has no other purpose, an example application is not included in this library.   Example code is in the Readme: https://github.com/PrimeGoose/f1-2021-udp#usage

**Comment by Joebel81** [comment]
Thank you so much for your answer. Please may I ask you kindly an example of 'usage' to receive the same datas i can see in to the folder 'Example-output'?
I'm not so good to create one myself.. sorry
Congratulation for this works 
Thanks
